lib/MusicBrainz/Server/Controller/WS/js/Edit.pm

```diff
@@ -578,6 +582,9 @@ sub edit : Chained('/') PathPart('ws/js/edit') CaptureArgs(0) Edit {
     unless ($c->user->has_confirmed_email_address) {
         $c->forward('/ws/js/detach_with_error', ['a confirmed email address is required']);
     }
+    if ($c->user->is_editing_disabled) {
+        $c->forward('/ws/js/detach_with_error', ['you are not allowed to enter edits']);
```

Comment:
This is not related to read-only mode, but was another thing not in sync with the regular edit pages.